onmt/utils/report_manager.py

```diff
@@ -129,11 +129,11 @@ def _report_training(self, step, num_steps, learning_rate, patience,
         report_stats.output(step, num_steps,
                             learning_rate, self.start_time)
 
-        self.maybe_log_tensorboard(report_stats,
-                                   "progress",
-                                   learning_rate,
-                                   patience,
-                                   step)
+        # self.maybe_log_tensorboard(report_stats,
+        #                            "progress",
+        #                            learning_rate,
+        #                            patience,
+        #                            step)
```

Comment:
?

Reply:
This provokes an inconsistent dispalying on tesorboard. Seems to be already done there
https://github.com/l-k-11235/OpenNMT-py/blob/7286ad98bf8df219ac009377e6cf04161e37d9d6/onmt/utils/report_manager.py#L156
https://github.com/l-k-11235/OpenNMT-py/blob/7286ad98bf8df219ac009377e6cf04161e37d9d6/onmt/utils/report_manager.py#L166

Reply:
Remove it then.